t/mojo/response.t

```diff
@@ -9,218 +9,231 @@ use Mojo::JSON qw(encode_json);
 use Mojo::Message::Response;
 use Mojo::Util qw(encode gzip);
 
-# Defaults
-my $res = Mojo::Message::Response->new;
-is $res->max_message_size, 2147483648, 'right default';
+# $dir is used in multiple subtests.
+my $dir = tempdir;
```

Comment:
It's only used in two places, that's not really worth sharing. The two tests can just have their own tempdir each.

Reply:
That's fine with me. I didn't want to potentially cause any changes in behavior by using two different temp dirs.